### DISCUSSION TITLE
Weird behaviour using UC

### DISCUSSION THREAD

**Discussion by HumbleExit** [seleniumbase/SeleniumBase#2472] [Q&A] [answered]
I've stumbled upon something I just can't come to understand. Anyone knows why this is happening? 

When I run an if statement in UC mode, Fiverr is able to detect a bot and blocks me from logging in. This doesn't happen when there is no if statement.

**Running this code Fiverr detects a bot**
```
from seleniumbase import SB
with SB(uc=True, test=True) as sb:
def loginFiverr():
    fiverrUsername = 'xxx'
    fiverrPassword = 'xxx'

    sb.driver.uc_open_with_tab("https://fiverr.com")
    if sb.is_element_present('nav-popover-items-toggler'):
        print("logged in already")
    else:
        sb.click("//a[contains(@href, '/login') and contains(text(), 'Sign in')]")
        sb.click("//section//button[p[contains(text(), 'Continue with email/username')]]")
        sb.type("#login", fiverrUsername, by="css selector")
        sb.type("#password", fiverrPassword, by="css selector")
        sb.click("//button[@type='submit'][p[text()='Sign In']]")
loginFiverr()
```

<img width="866" alt="fiverr-sc" src="https://github.com/seleniumbase/SeleniumBase/assets/98287617/811798ac-6cd2-4de3-a184-8d91f243859e">

**Running this code I can login without getting detected by Fiverr**
```
from seleniumbase import SB
with SB(uc=True, test=True) as sb:
    fiverrUsername = 'xxx'
    fiverrPassword = 'xxx'

    def loginFiverr():
            sb.driver.uc_open_with_tab("https://fiverr.com")
            sb.click("//a[contains(@href, '/login') and contains(text(), 'Sign in')]")
            sb.click("//section//button[p[contains(text(), 'Continue with email/username')]]")
            sb.type("#login", fiverrUsername, by="css selector")
            sb.type("#password", fiverrPassword, by="css selector")
            sb.click("//button[@type='submit'][p[text()='Sign In']]")
    loginFiverr()
```

**Comment by mdmintz** [comment, marked as answer]
Even calling a simple command such as `is_element_present(selector)` can make your browser detectable because that puts Selenium in the browser (where it can be found). Looks like you may have already answered your own question, as you have a script that works when you don't include that line.

**Reply by HumbleExit** [reply]
Do you give consultation?  I need advice. What is your hourly rate for a 30 minute call?

**Comment by OpsecGuy** [comment]
It seems to be an easy task if you would like to bypass what's on the screenshot.
Even if you getting detections, why not just use selenium actions? Simply get XPATH of that web element and emulate pressing a mouse.
